fix(apple silicon): fixed various typos and product names (#6694)
scaleway/docs-content: pages/apple-silicon/how-to/create-runner.mdx

## Creating a runner

1. Click [Apple silicon](https://console.scaleway.com/asaas/servers) in the **Bare Metal** section of the side menu. The [Apple silicon creation page](https://console.scaleway.com/asaas/servers) displays.
    <Message type="note">

>     Should your account already contain Mac minis, a list of these will be displayed.

Reviewer: ```suggestion
    If your account already contains Mac minis, they appear in a list.
```
